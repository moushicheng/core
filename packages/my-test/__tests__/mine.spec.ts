import { data1, data2 } from './index'

describe('my test', () => {
  test('测试快照 data1', () => {
    expect(data1()).toMatchSnapshot({
      name: 'Jsoning',
      age: 26,
      time: '2020.1.1'
    })
  })

  test('测试快照 data3', () => {
    expect(data2()).toMatchSnapshot({
      name: 'Jsoning',
      age: 26,
      time: expect.any(Date) //用于声明是个时间类型，否则时间会一直改变，快照不通过
    })
  })
})
