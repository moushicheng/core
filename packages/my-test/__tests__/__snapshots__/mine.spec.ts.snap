// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my test 测试快照 data1 1`] = `
Object {
  "age": 26,
  "name": "Jsoning",
  "time": "2020.1.1",
}
`;

exports[`my test 测试快照 data3 1`] = `
Object {
  "age": 26,
  "name": "Jsoning",
  "time": Any<Date>,
}
`;
